src/corebluetooth/adapter.rs

```diff
@@ -17,9 +17,8 @@ pub struct Adapter {
 }
 
 pub(crate) fn uuid_to_bdaddr(uuid: &String) -> BDAddr {
-    BDAddr {
-        address: uuid.as_bytes()[0..6].try_into().unwrap(),
-    }
+    let b: [u8; 6] = uuid.as_bytes()[0..6].try_into().unwrap();
+    BDAddr::try_from(b).unwrap()
```

Comment:
One of refactors that didn't quite survive the rebase was this. I may be worth reviewing as I ended up applying a dirty fix so that macOS/iOS would still build